### PR TITLE
Issue:#200 - remove annoying console.log statements in the code

### PR DESCRIPTION
**Pull request by Abhid1239**
# Description
remove annoying console.log statements in the code

## screenshots
Isssue ID: #200

### CODE REVIEW

**Review by NoamGaash — APPROVED** [review]
Cool, thanks

**Comment by shootermv** [comment]
not sure why do you closed the PR...
if you want your code to be added to the project - 
you need to wait until conde maintainer will merge your code (which suppose to happen soon after approval)

**Comment by NoamGaash** [comment]
@shootermv seems like he made a new PR :confused: 
https://github.com/hasadna/open-bus-map-search/pull/206

**Pull request closed without merging**
